lectures/L13-slides.tex

```diff
@@ -184,6 +184,7 @@
 
 We've already examined the signature; the first line of the definition.
 
+%Here's another modifier that isn't used; you say it has static, but the code doesn't have it
```

Comment:
Months removed again and "static" mention removed.

--
Michael Gaisser
850-396-4535

On Sun, Aug 21, 2016 at 2:35 AM, Jeff Zarnett <notifications@github.com>
wrote:

> In lectures/L13-slides.tex
> <https://github.com/jzarnett/ece150/pull/2#discussion_r75590241>:
>
> > @@ -184,6 +184,7 @@
> >
> >  We've already examined the signature; the first line of the definition.
> >
> > +%Here's another modifier that isn't used; you say it has static, but the code doesn't have it
>
> Feel free to delete this in your PR :)
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/jzarnett/ece150/pull/2/files/4594d7261bea3d94b2deae8b63477404c046594d#r75590241>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AA7ukrC9HpOditirKqxid_AaoFuCP5Ohks5qh__TgaJpZM4JZLoa>
> .
>